rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1322,9 +1323,9 @@ def test_upgrade_db_36_to_37(user_data_dir):  # pylint: disable=unused-argument
     new_history_events = cursor.execute('SELECT * FROM history_events;').fetchall()
     for idx, entry in enumerate(old_history_events):
         if entry[11] is None:
-            assert entry[0:11] + ('none',) + entry[12:] == new_history_events[idx]
+            assert entry[0:11] + ('none',) == new_history_events[idx]
         else:
-            assert entry == new_history_events[idx]
```

Comment:
you completely ignored to check that base history events table and evm events info table are properly populated in this test. If you had ... you would have seen that this does not happen